src/package/nuspec/TestPlatform.Build.nuspec

```diff
@@ -8,7 +8,7 @@
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>Build tasks and targets for running tests with Test Platform</description>
-    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</licenseUrl>
+    <license>http://www.microsoft.com/web/webpi/eula/net_library_eula_enu.htm</license>
```

Comment:
Can you make sure the built packages will list the license correctly after this change? In its documentation, I couldn't find a way to provide a URL, maybe I missed it.

Reply:
I am looking into this.